proto/buf/alpha/registry/v1alpha1/admin.proto

```diff
@@ -87,6 +87,21 @@ message MonthlyUsage {
   uint32 total_method_types = 5;
 }
 
+// RepositoryUsage contains the total amount of message, enum and method types
+// for an repository within an organization.
+message RepositoryUsage {
+  // The id of the repository.
+  string repository_id = 1;
+  // The name of the repository.
+  string repository_name = 2;
+  // The id of the repository owner.
+  string owner_id = 3;
+  // // The name of the respository owner.
+  string owner_name = 4;
+  // The total number of message, enum and method types usage for the repository.
+  MonthlyUsage usage = 5;
```

Comment:
is this supposed to be monthly (i.e. having a specific month and year)?

Reply:
then I will suggest to flatten this to just include what you need

Reply:
ok, replaced `MonthlyUsage` with 
```
  // The total amount of message types for the month and year.
  uint32 total_message_types = 5;
  // The total amount of enum types for the month and year.
  uint32 total_enum_types = 6;
  // The total amount of method types for the month and year.
  uint32 total_method_types = 7;
  ```